BenchmarkTest00230.py

```
@@ -0,0 +1,17 @@
+#{fact rule=untrusted-deserialization@v1.0 defects=1}
+
+from django.http import HttpResponse
+import datetime
+
+
+# pickle tests
+
+def current_datetime(request):
+    # ruleid:avoid-insecure-deserialization
+    user_obj = b64decode(request.cookies.get('uuid'))
```

Ruff (0.13.1)

Undefined name `b64decode`

(F821)